youtube_dl/extractor/abc.py

```diff
@@ -105,7 +105,8 @@ def _real_extract(self, url):
 
 class ABCIViewIE(InfoExtractor):
     IE_NAME = 'abc.net.au:iview'
-    _VALID_URL = r'https?://iview\.abc\.net\.au/(?:[^/]+/)*video/(?P<id>[^/?#]+)'
+    _VALID_URL = r'https?://iview\.abc\.net\.au/(?:[^/]+/)*(?P<type>video|show)/(?P<id>[^/?#]+)'
```

Comment:
Must be separate extractor.

Reply:
I had it separate originally, but HD trouble getting the tests to like it. Will have another try.